blob/blob.go

```diff
@@ -16,44 +18,104 @@ import (
 	"github.com/celestiaorg/celestia-node/share"
 )
 
+//nolint:unused
 var errEmptyShares = errors.New("empty shares")
```

Comment:
Let's remove it if its unused

Reply:
want to confirm with @vgonkivs  first

Reply:
afb2293718e9d4413e7761dfe2c697cb83726a77